test/recipes/20-test_speed.t

```diff
@@ -86,7 +86,8 @@ ok(run(app(['openssl', 'speed', '-testmode', '-aead', '-evp', 'aes-128-gcm'])),
 
 SKIP: {
     skip "ASYNC/threads not supported by this OpenSSL build", 1
-        if disabled("async") || disabled("threads");
+        if disabled("async") || disabled("threads")
+           || config('target') =~ m|^nonstop|;
```

Comment:
not sure I follow here.  

The error provided was "async_jobs specified but async not supported"

Which by my read indicates that the nonstop build is created with no-async omitted (which is fine)

but ASYNC_is_capable() returns false, implying that nonstop has no working getcontext function

Shouldn't the fix here be to modify the nonstop config to just default to a no-async build?

Reply:
A more generic fix would be somehow to skip the test if we know `ASYNC_is_capable` is going to return false. If we could figure out how to do that then we won't have to play whack-a-mole for any platform where that is the case.